### DISCUSSION TITLE
Mill profiling with -j 0 ?

### DISCUSSION THREAD

**Discussion by Quafadas** [com-lihaoyi/mill#3101] [Q&A]
I've been struggling to get one of my builds working reliably, and so have been trying to introspect it. I see this in the terminal. 

```
worker@default-64c4bbfd8c-mksdw:~/simon/Scala-ILS-Lib$ mill show smithyapi.jvm.test
[#0] [1/1] show > [106/116] smithyapi.jvm.compile 
[#0] [info] compiling 424 Scala sources to /home/worker/simon/Scala-ILS-Lib/out/smithyapi/jvm/compile.dest/classes ...
```

Which looks to me, as though it's crashing before it finishes compilation. I can't seem to figure out why though. 

Something I've observed, is that if you watch the "mill-chrome-profile.json" file, I think it's being quite agrgessively overwritten. I suspect not by the same processes... 

Is that intentional? 

I also get the same, with mill-profile.json?

**Comment by hmf** [comment]
You may be having a Mill server running. Make sure you kill all (related) processes and use Mill only in interactive mode: `mill -i`. See if that helps. 

HTHs
